pkg/controller/cassandradatacenter/controller/datacenter.go

```diff
@@ -0,0 +1,24 @@
+package controller
```

Comment:
lets rename this to `metadata`, since its responsible for generating object metatdata values